src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/weekly/WeeklyAdherenceReportGenerator.java

```diff
@@ -137,7 +144,19 @@ public WeeklyAdherenceReport generate(AdherenceState state) {
         }
         int percentage = AdherenceUtils.calculateAdherencePercentage(ImmutableList.of(finalReport));
         
+        ParticipantProgressionState progression = IN_PROGRESS;
+        if (rowList.isEmpty() && nextDay == null) {
+            long na = AdherenceUtils.counting(eventReport.getStreams(), ImmutableSet.of(NOT_APPLICABLE));
+            long total = AdherenceUtils.counting(eventReport.getStreams(), EnumSet.allOf(SessionCompletionState.class));
+            if (na == total) {
+                progression = UNSTARTED;    
+            } else {
+                progression = DONE;
+            }
+        }
```

Comment:
Multiple filters here would be helpful to week out unstarted accounts.

Reply:
Changed to take multiple values. Also changed collections to be sets so duplicates are automatically removed from JSON arrays.